docs/topics/cache.txt

```diff
@@ -542,9 +542,8 @@ include the name of the active :term:`language<language code>` -- see also
 :ref:`how-django-discovers-language-preference`). This allows you to easily
 cache multilingual sites without having to create the cache key yourself.
 
-Cache keys also include the active :term:`language <language code>` when
-:setting:`USE_L10N` is set to ``True`` and the :ref:`current time zone
-<default-current-time-zone>` when :setting:`USE_TZ` is set to ``True``.
+Cache keys also include the :ref:`current time zone <default-current-time-zone>`
+when :setting:`USE_TZ` is set to ``True``.
```

Comment:
As far as I'm concerned, previously when `USE_L10N` was `True` and `USE_I18N` was `False` then cache keys included `settings.LANGUAGE_CODE`. I think we should add release notes that in such configuration cache keys are different from the keys generated by older versions of Django :thinking:

Reply:
> By reading the code, it appears to me that the cache key is internal implementation. Do you see a way that this change could cause a backwards incompatibility?

No, but the first request to any previously cached value will be a cache miss. Am I right? :thinking: so maybe it's worth to add a short note (like we did in e.g. 5cb3ed187b2).